plugins/modules/system/filesystem.py

```diff
@@ -144,6 +158,14 @@ def create(self, opts, dev):
             cmd = "%s %s %s '%s'" % (mkfs, self.MKFS_FORCE_FLAGS, opts, dev)
         self.module.run_command(cmd, check_rc=True)
 
+    def wipefs(self, dev):
+        wipefs = self.module.get_bin_path('wipefs', required=True)
+        if self.module.check_mode:
+            cmd = "%s --no-act --all '%s'" % (wipefs, dev)
```

Comment:
```suggestion
            cmd = [wipefs, "--no-act", "--all", dev]
```
Please use the array notation for new `run_command()` calls.

Reply:
ok, done